styles/partials/core/_base.scss

```diff
@@ -28,6 +28,10 @@ a {
 
 }
 
+input {
+  box-sizing: border-box;
```

Comment:
The lack of a css reset has messed with me a number of times — maybe we should consider adding some extremely simple resets?